stdlib/LinearAlgebra/test/schur.jl

```diff
@@ -193,4 +193,13 @@ end
     end
 end
 
+@testset "adjoint and transpose for schur (#40941)" begin
+    A = rand(3, 3)
+    B = schur(A', A)
+    C = B.left*B.S*B.right'
+    D = schur(adjoint(A), A)
```

Comment:
```suggestion
    D = schur(transpose(A), A)
```
otherwise it's just the same as `A'`.

Reply:
Oops, I had transpose in mind but typed out adjoint. Done.